packages/auth/src/access/service.test.ts

```diff
@@ -21,9 +22,9 @@ describe('Access Service', (): void => {
   let trx: Knex.Transaction
 
   beforeAll(async (): Promise<void> => {
-    deps = await initIocContainer(Config)
-    accessService = await deps.use('accessService')
+    deps = initIocContainer(Config)
     appContainer = await createTestApp(deps)
```

Comment:
order matters: if resolving a dependency before `createTestApp`, if will use a normal logger instead of a test logger, since `createTestApp` overrides the logger.